### PR TITLE
Text summary

### PR DESCRIPTION
**Pull request by francylang**
## What's this PR do?
Finally able to text all summary reports on the click of one button.
Also, added Beyonce to the roster

### CODE REVIEW

File: lib/components/Classroom/Classroom.js [hunk 1]
```diff
@@ -6,14 +6,19 @@ import textFetch from '../../utils/textFetch.js';
 
 
 
-const Classroom = ({ teacher, students, deleteStudent, userStatus, sendSummaries, coreValues, plusOneCoreValue }) => {
+const Classroom = ({ teacher, students, deleteStudent, userStatus, coreValues }) => {
+
+  function sendSummary(students, coreValues) {
```
Comment (ameseee) [inline lib/components/Classroom/Classroom.js]: what a great feature.

File: lib/containers/ClassroomContainer.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ export const mapStateToProps = store => ({
 });
 
 export const mapDispatchToProps = dispatch => ({
-  deleteStudent: (studentId) => dispatch(deleteStudent(studentId)),
```
Comment (ameseee) [inline lib/containers/ClassroomContainer.js]: why did you get rid of the trailing comma?!

File: lib/reducers/ClassroomReducer.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ export const initialState = {
     {id: 17, name: 'Luke Finney', parentContact: 'Sally Finney', phone: '444-333-3211'},
     {id: 18, name: 'Travis Haby', parentContact: 'Sally Haby', phone: '444-333-3211'},
     {id: 19, name: 'Christie Lynam', parentContact: 'Sally Lynam', phone: '444-333-3211'},
-    {id: 20, name: 'Hudson Lynam', parentContact: 'Sally Lynam', phone: '444-333-3211'}
+    {id: 20, name: 'Beyonce', parentContact: 'Sally Lynam', phone: '444-333-3211'}
```
Comment (ameseee) [inline lib/reducers/ClassroomReducer.js]: finally.

File: lib/components/Student/Student.js [hunk 4]
```diff
@@ -23,10 +23,6 @@ class Student extends Component {
   async sendSummary() {
     const studentId = parseInt(this.props.match.params.id);
     const student = this.props.students[studentId].name;
-    // const summaryValues = Object.values(this.props.coreValues)
```
Comment (jenPlusPlus) [inline lib/components/Student/Student.js]: getting rid of comments! 
giphy 7

**Pull request merged**